src/Gateways/Builtin/PayPalGateway.php

```diff
@@ -79,26 +79,29 @@ public function prepare(Prepare $data): Response
 
         return new Response(true, [
             'result' => [
-                'id' => $response->result->id,
+                'id'            => $response->result->id,
+                'currency_code' => $response->result->purchase_units[0]->amount->currency_code,
```

Comment:
> The PayPal SDK requires a currency code parameter, and I couldn't figure out how to get the current site's currency in the antlers template. What I've done (passing it into paypal.result) is very much a bodge. Hopefully you can correct this.

From what I can tell, this is what you're meaning - correct?

A couple of lines above this, we get the current site's currency with `Currency::get(Site::current())['code']`. You could probably do the same.

Reply:
Ah yeah, but I don't think that line really needs to be there at all, I only added it as a workaround so the currency code was available in the antlers template (in the `script` tag that loads the PP SDK, see the example in docs). If there's a way to get the current sites currency in antlers already this line could be removed.